### PR TITLE
Remove build artifacts and update build process

### PR DESCRIPTION
**Pull request by HammadTheOne**
This PR removes build artifacts from the `dev` branch of `dash` and updates the build scripts. It also includes some cleanup for obsolete html components. 

## Contributor Checklist

- [x] TODO
   -  [x] Update package.json scripts to a single `npm build` script to build renderer and components.
   -  [x] Update CONTRIBUTING.md
   -  [x] Update dash-core release process (https://github.com/plotly/dash-core/pull/336)
   -  [x] Remove obsolete `html` components.
   -  [x] Fix table helpers import issue (if in scope)

- [x] I have run the tests locally and they passed. (refer to testing section in [contributing](https://github.com/plotly/dash/blob/master/CONTRIBUTING.md))

### optionals

- [x] I have added entry in the `CHANGELOG.md`

### CODE REVIEW

**Review by jonmmease — APPROVED** [review]
I'll let @alexcjohnson give the final OK, but this looks good from my end.

Thanks @HammadTheOne!

File: CHANGELOG.md [hunk 1]
```diff
@@ -24,6 +26,11 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 
 - [#1729](https://github.com/plotly/dash/pull/1729) Include F#, C#, and MATLAB in markdown code highlighting, for the upcoming .NET and MATLAB flavors of dash.
 
+## Dash HTML Components
+### Changed
```
Comment (alexcjohnson) [inline CHANGELOG.md:30]:
```suggestion
### Removed
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -15,18 +15,15 @@ $ python3 -m venv .venv/dev
...
-$ cd dash-renderer
-# build renderer bundles, this will build all bundles from source code
-# the only true source of npm version is defined in package.json
 $ npm install
-$ npm run build  # or `renderer build`
-# install dash-renderer for development
-$ pip install -e .
+# this script will build the dash-core-components, dash-html-components, dash-table,
+# and renderer bundles; this will build all bundles from source code in their
+# respective directories.The only true source of npm version is defined
```
Comment (alexcjohnson) [inline CONTRIBUTING.md:21]:
```suggestion
# respective directories. The only true source of npm version is defined
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -15,18 +15,15 @@ $ python3 -m venv .venv/dev
...
+# in package.json for each package.
+$ npm run build  # runs `renderer build` and `npm build` in dcc, html, table
 # build and install components used in tests
-$ cd .. # should be back in dash/ root directory
-$ npm install
 $ npm run setup-tests.py # or npm run setup-tests.R
-# you should see both dash and dash-renderer are pointed to local source repos
+# you should see both dash is pointed to a local source repo
```
Comment (alexcjohnson) [inline CONTRIBUTING.md:26]:
```suggestion
# you should see dash points to a local source repo
```

**Review by alexcjohnson — APPROVED** [review]
💃 Love it! All sorts of good stuff here.

**Pull request merged**